### PR TITLE
feat(Homebrew): AND-1386 Render real data on order detail page

### PR DESCRIPTION
**Pull request by ditn**
screenshot1537440030

### CODE REVIEW

File: morph/ui/src/main/java/com/blockchain/morph/ui/homebrew/exchange/detail/HomebrewTradeDetailActivity.kt [hunk 1]
```diff
@@ -1,29 +1,160 @@
...
-        // TODO: data to test layout - use real object
         status.text = trade.state.toStatusString(this)
-        value.text = trade.depositQuantity
+        status.setCompoundDrawablesWithIntrinsicBounds(
+            trade.state.toDrawable(this),
+            null,
+            null,
+            null
+        )
+        value.text = if (trade.approximateValue()) "~${trade.price}" else trade.price
```
Comment (westonal) [inline morph/ui/src/main/java/com/blockchain/morph/ui/homebrew/exchange/detail/HomebrewTradeDetailActivity.kt]: Does this need localization?
Does `~` mean approx. in every locale?

File: morph/ui/src/main/java/com/blockchain/morph/ui/homebrew/exchange/detail/HomebrewTradeDetailActivity.kt [hunk 2]
```diff
@@ -1,29 +1,160 @@
...
-        fees.text = trade.price
+        fees.text = trade.fee
+
+        if (trade.shouldStrike()) {
+            receive.paintFlags = Paint.STRIKE_THRU_TEXT_FLAG or Paint.ANTI_ALIAS_FLAG
+        }
+
+        if (trade.refunding()) {
+            receive.text = "~${trade.quantity}"
+            fees.text = "~${trade.fee}"
```
Comment (westonal) [inline morph/ui/src/main/java/com/blockchain/morph/ui/homebrew/exchange/detail/HomebrewTradeDetailActivity.kt]: Whether localizable or not, I would at least extract a function for this `trade.fee.displayAsApproximate()`

File: morph/ui/src/main/java/com/blockchain/morph/ui/homebrew/exchange/detail/HomebrewTradeDetailActivity.kt (hunk 1 above)
Comment (ditn) [reply morph/ui/src/main/java/com/blockchain/morph/ui/homebrew/exchange/detail/HomebrewTradeDetailActivity.kt]: That's a really good question. It's a mathematical symbol so I would assume so but let me check.

Comment (ditn) [reply morph/ui/src/main/java/com/blockchain/morph/ui/homebrew/exchange/detail/HomebrewTradeDetailActivity.kt]: Doesn't look like it's something that's localised anywhere.

**Pull request merged**